draft-irtf-cfrg-hybrid-kems.md

```diff
@@ -679,6 +715,37 @@ def Decaps(dk, ct):
     return ss_H
 ~~~
 
+## Separate Key Generation
+
+The hybrid KEM framework described above prescribes a "shared seed" process for
+keypair generation, in which a shared seed is expanded into seeds for the
+individual constituent algorithms. This shared seed is used as the decapsulation
+key for the hybrid KEM and expanded into the two per-constituent decapsulation
+keys as necessary.
+
+In some deployment environments, it is not possible to instantiate this process.
```

Comment:
You could mention, for example, if your deployment environment requires compliance with NIST; specifically FIPS 203, SP 800-56, and/or SP 800-227.